ext/handlers/filters/message/message.go

```diff
@@ -47,6 +49,18 @@ func ForwardFromChatID(id int64) filters.Message {
 	}
 }
 
+func Regex(p string) filters.Message {
+	return func(m *gotgbot.Message) bool {
+		r, err := regexp.Compile(p)
```

Comment:
The compile should be moved outside of the anonymous `func()`, otherwise you recompile the regex every time and lose all the performance improvements from compiling it in the first place.

Reply:
Oh right, this way if we got an error from `regexp.Compile`, we can return it by `Regex` function
so the `Regex` function will become something like this `func Regex(p string) (filters.Message, error)`
this way users can handle it directly

Reply:
Right, Let me do so.